airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```
@@ -940,14 +940,54 @@ definitions:
         title: Cursor Datetime Formats
-        description: The possible formats for the cursor field, in order of preference. The first format that matches the cursor field value will be used to parse it. If not provided, the `datetime_format` will be used.
         type: array
         items:
           type: string
-          examples:
-            - "%Y-%m-%dT%H:%M:%S.%f%z"
-            - "%Y-%m-%d"
-            - "%s"
+        description: |
+          The possible formats for the cursor field, in order of preference. The first format that matches the cursor field value will be used to parse it. If not provided, the Outgoing Datetime Format will be used.
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)